tests/test_bin.py

```diff
@@ -75,8 +75,15 @@ def test_bin_hy_stdin_multiline():
 
 
 def test_bin_hy_history():
-    output, _ = run_cmd("hy", '(+ "a" "b")\n(+ *1 "y" "z")')
-    assert "'abyz'" in output
+    output, _ = run_cmd("hy", '(+ "a" "b")\n\
+                               (+ "c" "d")\n\
+                               (+ "e" "f")\n\
+                               (.format "*1: {}, *2: {}, *3: {}," *1 *2 *3)')
```

Comment:
Its a matter of style, but I prefer using brackets over `\`:

```python
output, _ = run_cmd("hy", (
    '(+ "a" "b")'
    '(+ "c" "d")'
    '(+ "e" "f")'
    '(.format "*1: {}, *2: {}, *3: {}," *1 *2 *3)'
))

Technically Hy doesn't care about the newlines either, so no need to have the visual noise.

Reply:
Thanks! Again, I honest didn't know the `run_cmd` could be used that way. This looks nicer ... but unfortunately guessing hy treat the tuple as 4 individual code pieces. So the `*1` ... is `None`.

I updated to `'''` -- hope it looks better.